Tests/UnitTests/Core/EventData/MultiTrajectoryTests.cpp

```diff
@@ -868,4 +868,19 @@ BOOST_AUTO_TEST_CASE(ProxyAssignment) {
   // MultiTrajectory::TrackStateProxy tp4{tp3};
```

Comment:
Did you check that this line still doesn't compile? (the compilation failure is what is expected).

Also did you test that `copyFrom` does **NOT** work when called on a const track state proxy?

Reply:
Yes, both lines still do not compile.

I also quickly tried to find a way to somehow turn such a compile error in a checkable `constexpr bool` but that seems not to be so easy unfortunately...

Reply:
Ok perfect!